ccm_web/client/src/pages/Home.tsx

```diff
@@ -26,39 +26,44 @@ function Home (): JSX.Element {
     title: 'Merge Sections',
     description: 'Combine sections into one Canvas site for easier management',
     icon: <MergeTypeIcon fontSize='large' />,
-    ordinalty: 1
+    ordinalty: 1,
+    route: '/merge'
```

Comment:
`merge-sections`? What do you think should be the route naming convention? I feel like I most often see hyphenation between words, definitely feels easier to read than it being all smushed together. `camelCase` would probably be fine, too.